Guide/jobs.markdown

```diff
@@ -44,11 +44,23 @@ instance Job EmailCustomersJob where
 
 ### Running the job
 
-IHP watches the job table in the database for any new records and automatically runs the job asynchronously when a new job is added. So to run a job, simply create a new record:
+IHP watches the job table in the database for any new records and automatically runs the job asynchronously when a new job is added. There are two ways to run a job:
+
+1. Run immediately:
 
 ```haskell
 newRecord @EmailCustomersJob |> create
 ```
+2. Schedule for future execution:
+
+```haskell
+now <- getCurrentTime
+newRecord @EmailCustomersJob 
+    |> set #runAt (addUTCTime 86400 now)  -- Schedule 24 hours in the future
+    |> create
```

Comment:
Let's use `nominalDay` day here instead of the hardcoded seconds

```haskell
import Data.Time.Clock (addUTCTime, getCurrentTime, nominalDay)

now <- getCurrentTime
newRecord @EmailCustomersJob 
    |> set #runAt (addUTCTime nominalDay now)  -- Schedule 24 hours in the future
    |> create
```